examples/deployment/scripts/2-gold-asc.js

```diff
@@ -0,0 +1,23 @@
+const {
+  printAssetHolding,
+  transferAsset,
+  asaOptIn,
+  asaOptInContract,
+  transferMicroAlgosContract
+} = require('../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
+
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
```

Comment:
This account is not used. Is it left for opt-in?

Reply:
Yes, when opt in works, then we will transfer some GOLD to this account

Reply:
let's move this line to the `gold_sc.js`

Reply:
ok